### ISSUE TITLE
Background image for boards

### ISSUE THREAD

**Issue by psy-q** [kanboard/kanboard#3357]
To better tell boards apart when working with a lot of them, it is useful to work with color or images. In Trello for example, you can define a background image (ideally something blurry) to go behind all the columns. It would be nice to have such a feature in Kanboard as well.

**Comment by rautamiekka** [comment]
If something is blurred, it should be done only once instead of rendered every single time. Besides, it probly adds to RAM usage, not to mention CPU usage.

**Comment by psy-q** [comment]
Ah yes, I just meant that many people choose an image that has been blurred (in e.g. Gimp) for the background, so it visually interferes less, but I wouldn't suggest doing the blurring inside the browser. Even though on many browser/GPU combinations that effect should be hardware-accelerated, but that's taking it way too far.

My idea would be similar to Trello, where you can upload any image and it is simply placed in the background and stretched to cover the whole area, aspect ratio permitting.

**Comment by fguillot** [comment]
Unfortunately, I do not plan to add such feature into Kanboard. I prefer to keep the software simple, the number of features is voluntary limited. The best approach is probably to write a custom plugin.

Thank you for your understanding
